webservices/common/views.py

```diff
@@ -32,10 +32,14 @@ class ApiResource(utils.Resource):
     def get(self, *args, **kwargs):
         query = self.build_query(*args, **kwargs)
         count = counts.count_estimate(query, models.db.session, threshold=500000)
+        multi = False
+        if isinstance(kwargs['sort'], (list, tuple)):
+            multi = True
+
         return utils.fetch_page(
             query, kwargs,
             count=count, model=self.model, join_columns=self.join_columns, aliases=self.aliases,
-            index_column=self.index_column, cap=self.cap,
+            index_column=self.index_column, cap=self.cap, multi=multi,
```

Comment:
@fec-jli this is a great catch in investigating and experimenting with what the issue was with the multi-sort not being supported when you were trying it out!  As I had mentioned when we caught up in PI planning, I know we've done this in a couple of other places and I had a chance to look into this some and now know why this appears to have not worked.

You're absolutely correct that you need to pass in the `multi` parameter with a value of `True`.  However, the fix shouldn't go here as this has ramifications across the whole API.  You have the right idea! :-)  We just need to adjust the where the change needs to go.

The changes in here should be reverted, and the fix (adjustment is more accurate) needs to go in the resource itself that you want to support mutli-sort in.  More details down below!

Reply:
Disregard this note!  Please see discussion below.